go.mod

```diff
@@ -2,22 +2,34 @@ module github.com/storacha/go-w3up
 
 go 1.23.3
 
+replace github.com/storacha/go-libstoracha => ../go-libstoracha
+
+replace github.com/storacha/go-ucanto => ../go-ucanto
```

Comment:
These ought to be removed.

Reply:
yes. As I mentioned in the "Dependencies" section in the PR's description, I still need to merge a couple things in that other repos. Once they are in, I'll point to the right versions.

The checks won't pass until I do that, so it's unlikely I'll forget doing it before merging 😄